automation/test-execution/ansible/tests/integration/conftest.py

```
@@ -0,0 +1,333 @@
+            subprocess.run(
+                [container_runtime, "stop", "-t", "2", container_id],
+                capture_output=True,
+                timeout=10,
+            )
+            subprocess.run(
+                [container_runtime, "rm", "-f", container_id],
+                capture_output=True,
+                timeout=5,
+            )
+        except Exception:
+            pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)